js/packages/cli/src/helpers/constants.ts

```diff
@@ -7,6 +7,8 @@ export const A = 'A';
 export const B = 'B';
 export const FEE_PAYER = 'fee_payer';
 export const TREASURY = 'treasury';
+export const CLAIM_STATE = 'claim';
+export const PURCHASE_STATE = 'purchase';
```

Comment:
Move these to js/packages/web/src/components/AuctionCard/types.ts and make it as enum.

enum ModalStage {
Claim,
}

Reply:
```
enum ModalStage {
Purchase,
Claim,
Hidden
}
```
To be consistent